app/models/annotations/status.rb

```diff
@@ -96,6 +96,7 @@ def destroy
       Annotation.find(self.id).destroy
     else
       widget.paper_trail.without_versioning do
+        widget.origin = self.origin
```

Comment:
Are we still using an `origin` that is passed from the controller all the way to the models? @caiosba ?